plugins/jobs/src/js/data/__tests__/JobModel-test.js

```diff
@@ -234,9 +234,9 @@ describe("JobModel Resolver", () => {
             jobsQuery({}, { path: [] }).pipe(take(1)),
             jobsQuery({}, { path: [] }).pipe(take(1)),
             jobsQuery({}, { path: [] }).pipe(take(1))
-          ).subscribe(jest.fn(), jest.fn(), () => {
-            expect(fetchJobs).toHaveBeenCalledTimes(1);
-          });
+          ).subscribe();
+          advance(20);
```

Comment:
that's cool!

Reply:
@DanielMSchmidt something we could try using in the data-layer extension plugin.